decidim-elections/app/models/decidim/votings/polling_officer.rb

```diff
@@ -6,18 +6,29 @@ class PollingOfficer < ApplicationRecord
       include Traceable
       include Loggable
 
-      belongs_to :user, foreign_key: "decidim_user_id", class_name: "Decidim::User", optional: true
+      belongs_to :user, foreign_key: "decidim_user_id", class_name: "Decidim::User"
```

Comment:
mmm, just to be sure, why it's not optional anymore? I can see you are checking for `user.nil?` below (I know it could be done before saving, but I'm not sure)

Reply:
As discussed offline, the `user` is always present, also when the user is invited on polling officer creation.